bluesky_widgets/_qt/qt_main_window.py

```diff
@@ -53,7 +53,8 @@ def __init__(self, qt_widget, *, show):
         self._qt_center = QWidget(self._qt_window)
 
         self._qt_window.setCentralWidget(self._qt_center)
-        self._qt_window.setWindowTitle(self.qt_widget.model.title)
+        if hasattr(self.qt_widget.model, 'title'):
+            self._qt_window.setWindowTitle(self.qt_widget.model.title)
```

Comment:
This title injection code was cribbed from napari, and may need a rethink for our use case (ours being a component library vs napari's an extensible application). This is fine for now.